yarn-project/noir-libs/noir-aztec/src/abi.nr

```diff
@@ -261,6 +261,10 @@ impl ContractStorageRead {
     fn hash(self) -> Field {
         dep::std::hash::pedersen_with_separator(self.serialize(), GENERATOR_INDEX__PUBLIC_DATA_READ)[0]
     }
+
+    fn empty() -> Self {
```

Comment:
Do we need `empty_contract_storage_read()` if we have empty here, seems like we should use the `empty` instead for clarity. 

In `call_public_function_with_packed_args` in the `context` we populate `contract_storage_read` with the empty ones.